tests/v2/validation/connection/workload.go

```diff
@@ -0,0 +1,22 @@
+package connection
```

Comment:
can we name this workloads instead of connection? we can then add daemonsets, stateful sets related tests in that package.